todoComments/todoComments.md

```diff
@@ -0,0 +1,185 @@
+| 55 | Using the following seems to have no impact on the code. It has been noticed that this function is called many times for each change. Someone should look at why that is happening and why some have no items in the arrays. | src/client/app/containers/MapChartContainer.ts | Needs Issue Created | - | size.length > 0 doesn't seem to be working as expected |
+| 56 | The max circle diameter should come from admin/DB. | src/client/app/containers/MapChartContainer.ts | Needs Issue Created | - | - |
+| 57 | Should be env variable? | src/client/app/redux/api/baseApi.ts | Unsure | - | - |
+| 58 | Validate Behavior against all endpoints | src/client/app/redux/api/baseApi.ts | Needs Issue Created | - | Currently no validation in place for the endpoints |
+| 59 | write more robust logic for error handling, and manually invalidate tags instead? | src/client/app/redux/api/conversionsApi.ts | Needs Issue Created | - | the deleteConversion function needs to have error handling and behavior verification, this is also true for the editConversion below it in the code |
+| 60 | Verify Behavior w/ Maintainers | src/client/app/redux/api/conversionsApi.ts | Related To An Issue | - | related to the error/handling and behavior verification |
+| 61 | write more robust logic for error handling, and manually invalidate tags instead? | src/client/app/redux/api/conversionsApi.ts | Related To An Issue | - | related to the error/handling and behavior verification |
+| 62 | Verify Behavior w/ Maintainers | src/client/app/redux/api/conversionsApi.ts | Related To An Issue | - | related to the error/handling and behavior verification |
+| 63 | Some of these can go away when we make the menus dynamic. | src/client/app/redux/selectors/adminSelectors.ts | Needs Issue Created | - | Didn't find any issues that mentioned this behavior in the admin section |
+| 64 | If a meter in a group is not visible to this user then it is not in Redux state and this fails. | src/client/app/redux/selectors/uiSelectors.ts | Needs Issue Created | - | No issues mentioning this problem, it seems that this section fails as certain meters in a group are not visible, leading to them not being in Redux state. |
+| 65 | Omitted for the time being re-implement slider range later. | src/client/app/redux/selectors/uiSelectors.ts | Issue Is Created | [Github issue #853](https://github.com/OpenEnergyDashboard/OED/issues/853) | I think that this TODO might be part of this already created issue, the actual slider range code is commented out and mentions that it needs to be re-implemented later, perhaps is the reason behind the issues with the slider range. |
+| 66 | Under development so in future | src/client/app/redux/slices/graphSlice.ts | Unsure | - | Confused by the 'Under Development', no mentions of any links, wasn't able to find anything in the issues both in the open or closed section for this.  There might be an issue that's related to this. |
```

Comment:
This involves getting a more user friendly default when no time range is selected by the user. It needs to be done at some point. FYI.